requirements.in

```diff
@@ -1,7 +1,6 @@
-Flask==1.1.1
+Flask>=1.0.4,<1.1.0
 Flask-Login>=0.4.1,<0.5.0
 Flask-WTF>=0.14.2,<0.15.0
-itsdangerous==1.1.0
```

Comment:
Just so it's clear - we haven't released this app with `itsdangerous==1.1.0`. This dependency was mistakenly bumped by PyUp in https://github.com/alphagov/digitalmarketplace-user-frontend/pull/216.

Reply:
Yeah, in fact itsdangerous isn't running on preview either, because the requirements for Docker images are taken from the `requirements.txt` file, not the `requirements.in` file, but only the `requirements.in` file was updated.